cypress/e2e/devpool.cy.ts

```diff
@@ -80,14 +80,69 @@ describe("DevPool", () => {
     cy.get('div[id="issues-container"]').children().should("have.length", 2);
   });
 
-  it("Display a message on rate limited", () => {
-    cy.intercept("https://api.github.com/repos/*/*/issues**", (req) => {
-      req.reply({
-        statusCode: 403,
+  describe("Display message on rate limited", () => {
+    it("Should display retry timeframe and login request with no tasks and no user", () => {
+      cy.intercept("https://api.github.com/user", (req) => {
+        req.reply({
+          statusCode: 403,
+          body: {},
+        });
+      }).as("getUser");
+      cy.intercept("https://api.github.com/repos/*/*/issues**", (req) => {
+        req.reply({
+          statusCode: 403,
+        });
+      }).as("getIssues");
+
+      cy.visit("/");
+      cy.get(".preview-header").should("exist");
+      cy.get(".preview-body-inner").should("include.text", "Please log in to GitHub to increase your GitHub API limits, otherwise you can try again at");
+    });
+
+    it("Should display retry timeframe with no tasks loaded and a logged in user", () => {
+      cy.intercept("https://api.github.com/user", (req) => {
+        req.reply({
+          statusCode: 200,
+          body: githubUser,
+        });
+      }).as("getUser");
+      cy.intercept("https://api.github.com/repos/*/*/issues**", (req) => {
+        req.reply({
+          statusCode: 403,
+        });
+      }).as("getIssues");
+
+      cy.visit("/");
+      cy.get(".preview-header").should("exist");
+      cy.get(".preview-body-inner").should("include.text", "You have been rate limited. Please try again at");
+    });
+
+    it("Should display a hard one-hour retry timeframe with no auth token available", () => {
+      const urlParams = `#error=server_error&error_code=500&error_description=Error getting user profile from external provider`;
+
+      cy.on("uncaught:exception", () => {
```

Comment:
I don't think so as far as I've read from the cypress docs, s/o, gpt and blogs, if I'm mistaken would you mind showing me an example? The only way I've been able to have the test pass is catching the error like that the GUI says that you can disable that behaviour but I didn't think that was the way to go.

I could validate that the thrown error matches the error text but the toaster text is validation of that.

Reply:
I can give a look later. But if that is the only way, make sure to reset this so we don't silence all the errors from the tests, only this specific one.

Reply:
I think this should be removed, as well as the `throw` inside the function `getNewSessionToken`. We should not make the page crash only because we could not retrieve the token, a `console.error` should suffice.

Reply:
remove both mate, the only test failing is the login one which you have a pull open for I'm sure, yeah?

Reply:
I believe it would be this one https://github.com/ubiquity/work.ubq.fi/pull/35
Let's merge it first so you can rebase and make sure everything works as expexted.

Reply:
same issue still but it's expected to happen from time to time is that right?

```
   1) DevPool
       User can log in:
     CypressError: `cy.type()` can only accept a string or number. You passed in: `undefined`
```